config/routes.rb

```diff
@@ -288,6 +288,11 @@
   get "admin/performance", to: "admin/performance/overview#show"
   namespace :admin do
     namespace :performance do
+      resources :email_schedules do
+        member do
+          get "delete", action: :delete
```

Comment:
why not standard http DELETE?

Reply:
Good catch. I had a confirmation delete page initially but then I realised it's not needed. I will change it to http DELETE.